spmd/compiler/aot_function_patch.py

```diff
@@ -0,0 +1,144 @@
+from functorch._src.aot_autograd import (
+    default_partition,
+    AOTConfig,
+    PytreeThunk,
+    create_aot_dispatcher_function,
+    KNOWN_TYPES,
+)
+from functools import wraps
+from typing import Callable, Dict, Optional, Tuple
+import torch.utils._pytree as pytree
+
+
+def patched_aot_function(
```

Comment:
what's this patched aot_function for? can we add a comment about why we need this?

Reply:
Added a comment .